packages/code-du-travail-frontend/pages/kali.js

```diff
@@ -25,7 +32,7 @@ class Kali extends React.Component {
 
   render() {
     const { data } = this.props;
-    if (data.status === 404) {
+    if (!data || data.status === 404) {
```

Comment:
We are to close to the elastic way. The front end should not be dependent on `data.status` nor `data._source`. 
We should have an abstraction between the data and how we present it.
IMO data should be transformed to be `data._source` if we have 4XX errors, we empty data (`= []`. If we data is empty we answer : `Cette convention collective n'a pas été trouvée`
\cc @lionelB

Reply:
agred but I'd rather do this refactor in a latter ticket though. ok with you? I'll create a ticket for this if you want.

Reply:
actually @douglasduteil it moved a bit because of another comment on the route. can you have a look at the new file and tell me if it suits you? https://github.com/SocialGouv/code-du-travail-numerique/pull/603/files#diff-7dff95c0cc72d9f01e6c85d6adc5a934